documentation/airport-guides/lkpr.md

```diff
@@ -0,0 +1,66 @@
+### About Prague
+
+Mezinárodní letiště Václava Havla Praha (Václav Havel Airport Prague) (ICAO: LKPR, IATA: PRG) is the main and biggest airport in the Czech Republic, acting as the hub for:
+- Czech Airlines (ICAO: CSA, callsign: CSA)
+- Travel Service and their subsidiary SmartWings (ICAO: TVS, callsign: SKYTRAVEL)
+- Ryanair (ICAO: RYR, callsign: RYANAIR)
+- Wizz Air (ICAO: WZZ, callsign: WIZZAIR),
+
+And a focus city for: 
+- easyJet (ICAO: EZY, callsign: EASY)
+- Vueling (ICAO: VLG, callsign: VUELING)
+
+It serves the capital city and most of the area west of Prague. In the east it is helped by Brno Tuřany (LKTB), Pardubice Airport (LKPD) and Ostrava Mošnov (LKMT). It is located 7NM (13 km) to the North-West of the city centre. 
+
+The Czech civil aviation history started at Prague Kbely Airport in 1918, but it's capacity was insufficient by the mid 1930s the Czech government decided to build a new airport in Prague-Ruzyně. Throughout the years, the airport went through many developmental changes, mainly the construction of runways 04/22, 06/24 and 12/30. 
+
+### Statistics
+
+15,415,001 passengers in 2017
+148,283 aircraft movement in 2017
+
+[Source](https://en.wikipedia.org/wiki/V%C3%A1clav_Havel_Airport_Prague)
+
+### Airspace
+
+The airspace is mainly a circle with a radius of 35nm situated around the Václav Havel Airport with two extended pieces of the airspace to the South-East and South-West. It stretches from ground level to FL145, although departures are permitted to climb past that altitude. 
+
+### Standard Operations
+
+###### Runway usage
+
+The Václav Havel Airport is very specific with it's runway usage. RWY04/22 is permanently out of operation and serves as a taxiway for oversized aircraft such as the Antonov AN-225. It is also for this reason not at all included in openScope. RWY24 is the only runway approved for ILS CAT II/III operations, runways 06, 12 and 30 are ILS CAT I certified. 
+
+The monopoly in CAT III certification and winds coming mostly from the west mean RWY24 is the preffered RWY and is used for approximately 80% of all operations. RWY12 is the second most commonly used runway and RWYs 06 and 30 are used very seldom. RWYs 06/24 and 12/30 are <b>never</b> used simultaneously. 
+
```

Comment:
typo: preferred, not preffered

Reply:
replace `<b>` with `**` and `</b>` with `**`

Reply:
If you wish you can quote from the AIP:

RWY in use will be determined in following order:
RWY 24
RWY 06
RWY 30
RWY 12